crates/turborepo-ffi/src/lockfile.rs

```diff
@@ -37,17 +39,44 @@ pub extern "C" fn npm_transitive_closure(buf: Buffer) -> Buffer {
     .into()
 }
 
-fn npm_transitive_closure_inner(buf: Buffer) -> Result<proto::LockfilePackageList, Error> {
+fn transitive_closure_inner(buf: Buffer) -> Result<proto::WorkspaceDependencies, Error> {
     let request: proto::TransitiveDepsRequest = buf.into_proto()?;
-    let lockfile = NpmLockfile::load(request.contents.as_slice())?;
-    let transitive_deps =
-        transitive_closure(&lockfile, request.workspace_dir, request.unresolved_deps)?;
-    let list: Vec<_> = transitive_deps
-        .into_iter()
-        .map(proto::LockfilePackage::from)
-        .collect();
+    match request.package_manager.as_str() {
+        "npm" => npm_transitive_closure_inner(request),
```

Comment:
Does it make sense to use a proto enum here instead (https://protobuf.dev/programming-guides/proto3/#enum)?